### DISCUSSION TITLE
Crater recurring invoices not generating

### DISCUSSION THREAD

**Discussion by happy4git2be** [crater-invoice-inc/crater#823] [Q&A]
Team - thank you in advance for your work crater.  It is a great tool.

My question:  I created Recurring Invoices.  I set them the Frequency to First Day of Month.  They did not create or email.  I am looking for guidance on what I did wrong.

I can create an invoice on demand under Invoices and send it successfully.

I have searched these discussions, the crater user documentation and done searches online but have not found an answer.

BACKGROUND:
 I recently installed crater on Ubuntu 20.04 LTS.  I installed it using docker-compose.

Crater is running well.  I can create and invoice and send via email.

Thank you for your time.

**Comment by lucacastelnuovo** [comment]
Hi, I'm not a developer of this project. 

This sounds like CRON isn't configured properly, with the consequence of not triggering the 'artisan schedule:run' command.

Can you validate the cronjob is executing properly?

**Reply by ppmt** [reply]
hello,
I am having the same issue.  I am also using docker-compose

How can I check that the cronjob is properly executed?
